website/docs/releases/linux.md

```diff
@@ -27,7 +27,7 @@ for now). They depend on the following packages:
 
 ### Install dependencies on Ubuntu and Debian
 
-Ubuntu 18.04 or newer, and Debian 9 or newer is required.
+Ubuntu 20.04 or newer, and Debian 10 or newer is required.
```

Comment:
_Debian 10 or newer_ -> _Debian 11 or newer_

Reply:
My thinking was this is for the current website, so for the current 0.81.x version, which did support Debian 10. Although we're dropping support for Debian 10 right before the 0.81.1 patch release, so probably best to update this to Debian 11 then? Ubuntu 20.04 can stay, or upgrade that one as well?

Reply:
I'm not sure what is the process of updating the website, but I would prefer to have up-to-date information in the documentation in our repository.

Reply:
Ok this PR is against main actually, so that's for 0.82.x. 

At a high level, 0.82.x -> Deb 11 minimum, 0.81.x -> Deb 10 minimum, is this correct?

Reply:
Ok, upgrading to Deb 11 as this is the 0.82.x stream (main).